app/src/main/res/values/dimens.xml

```diff
@@ -15,6 +15,7 @@
     <dimen name="margin_padding_size_20">20dp</dimen>
     <dimen name="margin_padding_size_22">22dp</dimen>
     <dimen name="margin_padding_size_24">24dp</dimen>
+    <dimen name="margin_padding_size_26">26dp</dimen>
     <dimen name="margin_padding_size_32">32dp</dimen>
     <dimen name="margin_padding_size_34">34dp</dimen>
     <dimen name="margin_padding_size_36">36dp</dimen>
```

Comment:
We should really improve the naming, before everyone starts using random dimens xD. I have no clue what margins are used right now. Can you do it, and remove the rest?

Reply:
idea is to use this instead make hardcode "20 dp" for example.